build/config/linux/cross_build.gni

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2021 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+declare_args() {
+  sysroot = ""
+  target_triple = ""
```

Comment:
Can we set this based on current_cpu instead?

See https://github.com/project-chip/connectedhomeip/pull/9353

Reply:
Removed this varible, as the only user is to specify the `system_libdir`.